easybuild/easyconfigs/h/HiC-Pro/HiC-Pro-2.9.0-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,66 @@
+easyblock = 'MakeCp'
+
+name = 'HiC-Pro'
+version = '2.9.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://nservant.github.io/HiC-Pro'
+description = """HiC-Pro was designed to process Hi-C data, from raw fastq files (paired-end Illumina data) 
+ to the normalized contact maps."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://github.com/nservant/HiC-Pro/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = [
+    'b82f15747bce937c3337b8120ad331a4bc07bbed6708d1d38b7d6ff1047a7a81',  # v2.9.0.tar.gz
+    '7006902cff8367b6bf1ef8cabd4de26bdbfa842062a0250755ff703227e88f34',  # HiC-Pro.patch
+]
+
+exts_defaultclass = 'PythonPackage'
+
+patches = ['HiC-Pro.patch']
```

Comment:
@timeu Please rename the patch to something like `HiC-Pro-2.9l0_eb.patch`, and move this up between `sources = ` and `checksums =`